DESCRIPTION

```diff
@@ -1,89 +1,88 @@
-Type: Package
 Package: mrcommons
+Type: Package
 Title: MadRat commons Input Data Library
-Version: 1.51.6
-Date: 2025-03-28
-Authors@R: c(
-    person("Benjamin Leon", "Bodirsky", , "bodirsky@pik-potsdam.de", role = "aut"),
-    person("Kristine", "Karstens", role = "aut"),
-    person("Lavinia", "Baumstark", role = "aut"),
-    person("Isabelle", "Weindl", role = "aut"),
-    person("Xiaoxi", "Wang", role = "aut"),
-    person("Abhijeet", "Mishra", role = "aut"),
-    person("Stephen", "Wirth", role = "aut"),
-    person("Mishko", "Stevanovic", role = "aut"),
-    person("Nele", "Steinmetz", role = "aut"),
-    person("Ulrich", "Kreidenweis", role = "aut"),
-    person("Renato", "Rodrigues", role = "aut"),
-    person("Roman", "Popov", role = "aut"),
-    person("Florian", "Humpenoeder", role = "aut"),
-    person("Anastasis", "Giannousakis", role = "aut"),
-    person("Antoine", "Levesque", role = "aut"),
-    person("David", "Klein", role = "aut"),
-    person("Ewerton", "Araujo", role = "aut"),
-    person("Eva", "Bleidorn", role = "aut"),
-    person("Felicitas", "Beier", role = "aut"),
-    person("Julian", "Oeser", role = "aut"),
-    person("Michaja", "Pehl", role = "aut"),
-    person("Debbora", "Leip", role = "aut"),
-    person("Michael", "Crawford", role = "aut"),
-    person("Edna", "Molina Bacca", role = "aut"),
-    person("Patrick", "von Jeetze", role = "aut"),
-    person("Eleonora", "Martinelli", role = "aut"),
-    person("Felix", "Schreyer", role = "aut"),
-    person("Bjoern", "Soergel", role = "aut"),
-    person("Pascal", "Sauer", role = "aut"),
-    person("David", "Hötten", role = "aut"),
-    person("Robin", "Hasse", role = "aut"),
-    person("Gabriel", "Abrahão", role = "aut"),
-    person("Pascal", "Weigmann", role = "aut"),
-    person("Jan Philipp", "Dietrich", , "dietrich@pik-potsdam.de", role = c("aut", "cre"))
-  )
-Description: Provides useful functions and a common structure to all the
-    input data required to run models like MAgPIE and REMIND of model
-    input data.
-License: LGPL-3 | file LICENSE
-URL:https://github.com/pik-piam/mrcommons,
-    https://doi.org/10.5281/zenodo.3822009
-BugReports: https://github.com/pik-piam/mrcommons/issues
+Version: 1.51.0
```

Comment:
Looks like sth went wrong with buildLibrary. Why did the version decrease from 1.51.6 -> 1.51.0?

Reply:
Weird.